Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM node:lts-slim
```

Comment:
It's better to specify the specific version of `node`.

```suggestion
FROM node:18-slim
```

Reply:
Critical seems to support Node 18 and Node 20. I would then prefer/recommend to set the version to 20.